bin/reth/src/config.rs

```diff
@@ -1,15 +1,45 @@
-//! CLI command to write default config to stdout
+//! CLI command to show configs
 use clap::Parser;
+use eyre::{bail, WrapErr};
 use reth_staged_sync::Config;
 
+use crate::dirs::{ConfigPath, PlatformPath};
+
 /// `reth config` command
 #[derive(Debug, Parser)]
-pub struct Command;
+pub struct Command {
+    #[clap(subcommand)]
+    command: Subcommands,
+}
+
+/// `reth config` subcommands
+#[derive(Debug, Parser)]
+pub enum Subcommands {
+    /// Show the default config
+    Default,
+    /// Show the active config
+    Show {
+        /// The path to the configuration file to use.
+        #[arg(long, value_name = "FILE", verbatim_doc_comment, default_value_t)]
+        config: PlatformPath<ConfigPath>,
+    },
+}
```

Comment:
hmm,
I'm not convinced the subcommand is the best UX here.

looks like we have three cases:

```
reth config # print current active config
reth config <file>
reth config --default
```

so I'd like to get rid of the subcommand here and use mutually exclusive arguments, which would require:

1. `config: Option<PlatformPath>`
2. `default: bool` `conflicts_with = "config"`

Reply:
cool, done :+1: